docs/documentation/home.md

```diff
@@ -1,5 +1,7 @@
 <!-- Links in /docs/documentation should NOT have `.md` at the end, because they end up in our wiki at release. -->
 
+**Documentation below is for CLI version 6. For version 7 see [here](https://angular.io/cli)**.
+
 # Angular CLI
 
 NOTE: this documentation is for Angular CLI 6. For Angular CLI 1.x go [here](1-x-home) instead.
```

Comment:
nit: this page already had a redirect for 6 vs 1 so maybe it should now have the nav to those three versions all together?